pkg/katatestutils/constraints.go

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package katatestutils
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// Operator represents an operator to apply to a test constraint value.
+type Operator int
+
+const (
+	TestDisabledNeedRoot    = "Test disabled as requires root user"
+	TestDisabledNeedNonRoot = "Test disabled as requires non-root user"
+
+	equalOperator    Operator = iota
+	notEqualOperator Operator = iota
+
+	osRelease = "/etc/os-release"
+
+	// Clear Linux has a different path (for stateless support)
```

Comment:
nit - The use of `/usr/lib/os-release` is part of the os-release spec, so may not only be Clear specific :-)
https://www.freedesktop.org/software/systemd/man/os-release.html